atk.yml

```diff
@@ -78,7 +78,7 @@ services:
         ROS_DISTRO: "humble"
         USER_UID: "@{uid}"
         USER_GID: "@{gid}"
-        APT_DEPENDENCIES: "bash zsh vim git git-lfs python3-pip python3-tk python3-opencv rapidjson-dev ros-humble-usb-cam ros-humble-tf-transformations"
+        APT_DEPENDENCIES: "bash zsh vim git git-lfs python3-pip python3-tk python3-opencv rapidjson-dev ros-humble-usb-cam ros-humble-tf-transformations ros-humble-rqt*"
```

Comment:
why install all of them? maybe just install ros-humble-rqt-graph?

Reply:
Fixed in latest push.